styles/Login/login.css

```diff
@@ -55,4 +63,13 @@
   justify-content: flex-start;
   padding: 4px 0;
   font-size: 0.9rem;
-}
\ No newline at end of file
+}
+
+.logout-container .img-container{
+  max-width: 30rem;
+}
+
+.img-container img{
+  border-radius: 10px;
```

Comment:
here too, use rem, be consistent with the units you use throughout your project

Reply:
Done.